### DISCUSSION TITLE
Where update

### DISCUSSION THREAD

**Discussion by Aloneinthedarkwed** [thatskymod/Sky-CotL-Scripts#140] [Q&A] [answered]
Where lib update TT

**Comment by thatskymod** [comment, marked as answer]
Thanks for reaching out! Let's go through a few steps to ensure your libs are up-to-date.

1. Update Sky to version [0.27.0](https://thatgamecompany.helpshift.com/hc/en/17-sky-children-of-the-light/faq/1356/)
2. Update **Canvas** to the newly released version [1.5.6](https://github.com/thatskymod/Sky-CotL-Scripts/blob/754d38203696c8feea8dff40c01f492befc6d85f/Canvas/Canvas.apk)
3. Update **libLangit** to the—also newly released—version [0.4.4](https://github.com/thatskymod/Sky-CotL-Scripts/blob/05fa329f2e50de2d33b6232b5005705f45e5ac91/Canvas/Libs/libLangit044.so).

This combination should resolve the immediate crashing.

Detailed change lists can be found at [Langit Discussion Board](https://t.me/dukunlangit/8/40577), the Canvas [releases page](https://github.com/RomanChamelo/Canvas-Open-Source/releases/latest) and thatgamecompany's [patch notes](https://thatgamecompany.helpshift.com/hc/en/17-sky-children-of-the-light/faq/1356/).
